microsoft-edge/devtools-guide-chromium/javascript/snippets.md

```diff
@@ -22,149 +22,158 @@ ms.date: 05/04/2021
 
-### Open the Snippets tab with a mouse
+1. To open DevTools, right-click the webpage, and then select **Inspect**. Or, press **Ctrl+Shift+I** (Windows, Linux) or **Command+Option+I** (macOS). DevTools opens.
 
-1. Select the **Sources** tab.  The **Sources** tool appears:
+1. In DevTools, on the Activity Bar, select the **Sources** tab. If that tab isn't visible, click the **More tools** (![More tools icon](./snippets-images/more-tools-icon.png)) button.
 
-   ![The Sources tool with the Page tab open on the left](./snippets-images/javascript-sources-page-pane.png)
+   ![The Sources tool with the Page tab open on the left](./snippets-images/sources-tool-page-pane.png)
 
 1. In the **Navigator** pane (on the left), select the **Snippets** tab.  To access the **Snippets** option, you might need to click the **More tabs** (![More tabs](./snippets-images/more-tabs-icon.png)) button.
 
-### Open the Snippets tab with the Command Menu
+#### Open the Snippets tab with the Command Menu
+
+You can also open the **Snippets** tab with the Command Menu:
 
 1. Select anything in DevTools, so that DevTools has focus.
 
 1. Press **Ctrl+Shift+P** (Windows, Linux) or **Command+Shift+P** (macOS) to open the Command Menu.
 
-1. Type **Snippets**, select **Show Snippets**, and then press **Enter** to run the command:
+1. Start typing "Snippets", select **Show Snippets**, and then press **Enter** to run the command:
 
-   ![The Show Snippets command](./snippets-images/javascript-search-show-snippets.png)
+   ![The Show Snippets command](./snippets-images/command-menu.png)
 
 
 <!-- ====================================================================== -->
-## Create Snippets
+## Create new snippets
 
-### Create a Snippet through the Sources tool
+To create a new snippet from the **Snippets** tab:
 
 1. [Open the Snippets tab](#open-the-snippets-tab).
 
 1. Click **New snippet**.
 
-1. Enter a name for your Snippet, and then press **Enter**:
+1. Enter a name for your snippet, and then press **Enter**:
 
-   ![Name a Snippet](./snippets-images/javascript-sources-snippets-naming.png)
+   ![A new, empty, snippet in the Sources tool](./snippets-images/new-snippet.png)
 
-### Create a Snippet through the Command Menu
+#### Create a new snippet with the Command Menu
 
 1. Focus your cursor somewhere in DevTools.
 
 1. Press **Ctrl+Shift+P** (Windows, Linux) or **Command+Shift+P** (macOS) to open the Command Menu.
 
-1. Start typing **snippet**, select **Create new snippet**, and then press **Enter**:
+1. Start typing "snippet", select **Create new snippet**, and then press **Enter**:
 
-   ![The command for creating a new Snippet](./snippets-images/javascript-search-create-new-snippet.png)
+   ![The command for creating a new snippet](./snippets-images/new-snippet-command-menu.png)
 
-To rename your new Snippet with a custom name, see [Rename Snippets](#rename-snippets).
+To rename your new snippet, see [Rename snippets](#rename-snippets).
```

Comment:
```suggestion
To rename your new snippet, see [Rename snippets](#rename-snippets), below.
```
Present section rendered:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/javascript/snippets?branch=pr-en-us-2876#create-a-new-snippet-with-the-command-menu